forwarder/writer.go

```diff
@@ -104,19 +139,27 @@ func (f *Forwarder) Write(ctx context.Context, req *remote.WriteRequest) (*remot
 		}
 	}
 
+	wg.Add(len(toWrite))
 	// Write each batch to the Vulcan bus.
 	for key, wr := range toWrite {
-		ll = ll.WithFields(log.Fields{"key": key})
 
-		ll.WithFields(log.Fields{
-			"timeseries_count": len(wr.Timeseries),
-		}).Debug("writing to bus")
+		go func(key string, wr *remote.WriteRequest) {
```

Comment:
I'd like this to use a worker pool of already available goroutines communicating over a work channel vs spawning a new goroutine per batch.

e.g. https://github.com/digitalocean/vulcan/blob/master/cassandra/writer.go#L120-L131